cpp/CMakeLists.txt

```diff
@@ -6,16 +6,31 @@ if(WIN32)
   add_compile_options(/DNOMINMAX)
 endif()
 
-add_subdirectory(cluster)
-add_subdirectory(density)
-add_subdirectory(diffraction)
-add_subdirectory(environment)
-add_subdirectory(locality)
-add_subdirectory(order)
-add_subdirectory(parallel)
-add_subdirectory(pmft)
-add_subdirectory(util)
+# Detect when building against a conda environment set the _using_conda variable
+# for use both in this file and in the parent
+get_filename_component(_python_bin_dir ${PYTHON_EXECUTABLE} DIRECTORY)
+if(EXISTS "${_python_bin_dir}/../conda-meta")
+  message("-- Detected conda environment, setting INSTALL_RPATH_USE_LINK_PATH")
+  set(_using_conda On)
+  set(_using_conda
+      On
+      PARENT_SCOPE)
+else()
+  set(_using_conda Off)
+  set(_using_conda
+      Off
+      PARENT_SCOPE)
+endif()
```

Comment:
```suggestion
```
I don't think this is necessary. I removed similar blocks from gsd, fresnel, and hoomd a long time ago.